.github/workflows/gh-sync.yml

```diff
@@ -0,0 +1,34 @@
+name: Sync GitHub with ADO
+
+on: 
+  issues:
+    types: [closed, edited, deleted, reopened, assigned, unassigned, labeled, unlabeled]
+  issue_comment:
+
+jobs:
+  sync-issues:
+    name: Run gh-sync from GitHub action
+    if: ${{ github.event.label.name == 'tracking' || contains(github.event.issue.labels.*.name, 'tracking') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Azure
+        uses: Azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - id: AzureKeyVault
+        uses: Azure/get-keyvault-secrets@v1
+        with:
+          keyvault: 'kv-qdk-build'
+          secrets: 'ghSyncBuildPAT'
+
+      - name: 'Trigger gh-sync'
+        uses: microsoft/gh-sync@main
+        with:
+          ado-organization-url: ${{ secrets.ADO_URL }}
+          ado-project: ${{ secrets.ADO_PROJECT }}
+          ado-area-path: ${{ secrets.ADO_AREA_PATH }}
+          github-repo: 'microsoft/jupyter-core'
+          issue-number: ${{ github.event.issue.number }}
+          ado-token: ${{ steps.AzureKeyVault.outputs.ghSyncBuildPAT }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
[nit]
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}

```

Reply:
Pesky spaces! How are you noticing them in the yaml files?

Reply:
The GitHub review UI gives a special warning for any file that doesn't end with a trailing newline...